app/views/content/terms_candidate.md

```diff
@@ -43,3 +56,21 @@ All candidates must have an enhanced DBS check. This will show up any spent or u
 Not all convictions will stop you from teaching. [Talk to your provider about their policy on criminal convictions](https://www.gov.uk/exoffenders-and-employment).
 
 Your name will also be checked against a DBS list of people who have been barred from working with children.
+
+## Questions or support
+
+If you need our help, email <becomingateacher@digital.education.gov.uk>.
+
+We respond within 5 working days, or 1 working day for more urgent queries.
+
+If you’re applying from overseas, you can find out more about becoming a teacher in England in our guidance for overseas graduates.
+
+## Changing our service
+
+Our service is always evolving, and we will make changes from time to time. We won’t tell you about the changes beforehand. We will let you know if the website will be unavailable while we’re updating it.
+
+## About your online account
+
+When you start your application, we’ll create an account for you automatically. Any information you give will be stored there. We’ll keep your data safe and only use it in line with our [privacy policy](https://www.apply-for-teacher-training.service.gov.uk/candidate/privacy-policy).
+
+You don’t need a password for your account. Instead, each time you log in, we’ll email you a unique and secure link.
```

Comment:
We should avoid negative contractions for consistency.

Change 'don't' to 'do not'